lib/spark.js

```diff
@@ -149,6 +150,16 @@ export default class Spark {
       console.error(err)
     }
 
+    if (stats.timeout) {
+      Zinnia.activity.error('Retrieval timed out')
+    } else if (stats.carTooLarge) {
+      Zinnia.activity.error('CAR too large')
+    } else if (stats.statusCode >= 400) {
+      Zinnia.activity.error('Bad response')
+    } else {
+      Zinnia.activity.info('Retrieval successful')
+    }
+
     const measurementId = await this.submitMeasurement(retrieval, { ...stats })
```

Comment:
Let's report an activity for submitting the measurement, too. `submitMeasurement` throws when the HTTP request fails.

https://github.com/filecoin-station/spark/blob/4c27d6b72982acc02ec3a98fd6fe04b23b9e7cc7/lib/spark.js#L121